guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,8 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing post-upgrade tasks
 
+* Optional: If you are using the {oVirt} compute resource, you can pre-migrate your data by executing the `foreman-rake ovirt:drop` task on your {ProjectServer}.
+The {oVirt} compute resource will be officially phased out in the 3.16 release and this migration will be triggered automatically during the upgrade to 3.16.
```

Comment:
I think that mentioning a particular release is okay for upstream and we know the release version for sure.

Reply:
I still wouldn't because we have downstream consumers. Katello users may also be more used to seeing those version numbers so if anything, I'd use `Foreman 3.16`.